### PR TITLE
MERC-758 - Fix paths for templates on Windows

### PR DESCRIPTION
**Pull request by mickr**
MERC-758

* Template-assembler fix for windows

### CODE REVIEW

**Comment by mickr** [comment]
@mcampa @sherrybc @bc-ejoe

File: lib/stencil-bundle.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ Bundle.prototype.assembleTemplatesTask = function (callback) {
             return upath.toUnix(file.replace(self.templatesBasePath + Path.sep, '').replace('.html', ''));
         });
 
-        Async.map(files, TemplateAssembler.assemble, function (err, results) {
+        Async.map(files, TemplateAssembler.assemble.bind(this, {bundle: true}), function (err, results) {
```
Comment (mcampa) [inline lib/stencil-bundle.js]: This does not look too clean. I would prefer to have to methods, `TemplateAssembler.assemble` and TemplateAssembler.assembleAndBundle`

**Comment by mickr** [comment]
http://mickryan-1465237113.my-integration.zone/1-l-le-parfait-jar/  Integration store running a Windows bundled theme.

**Comment by mickr** [comment]
http://g.recordit.co/AUUxpWFIMZ.gif

**Comment by mcampa** [comment]
👍

**Comment by mickr** [comment]
Stencil CLI running on Mac OS X

**Pull request merged**